### PR TITLE
Update participant count

### PR DESCRIPTION
**Pull request by andrewindechemain**
This pull request:
-Removes participant count and glyph from the join conference button

### CODE REVIEW

File: common/components/common/event_projects/JoinConferenceButton.jsx [hunk 1]
```diff
@@ -32,10 +32,6 @@ class JoinConferenceButton extends React.PureComponent<Props> {
         >
           <i className={Glyph(GlyphStyles.Video, GlyphSizes.LG) + " mr-3"} />
           {this.props.children}
-          <i className={Glyph(GlyphStyles.Users, GlyphSizes.LG) + " ml-3"} />
-          <span className="JoinConference-usercount overline ml-1">
-            {this.props.participant_count || "0"}
-          </span>
```
Comment (marlonkeating) [inline common/components/common/event_projects/JoinConferenceButton.jsx:38]: This looks like the only applicable change related to this feature.  Is there something I'm missing with the other changes?

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: common/components/componentsBySection/CreateEvent/EventPreviewForm.jsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ class EventPreviewForm extends React.PureComponent<Props> {
       <React.Fragment>
         <DjangoCSRFToken />
         <input type="hidden" name="is_created" value="True" />
-        <AboutEventDisplay event={this.props.project} viewOnly={true} />
+        <AboutEventDisplay event={this.props.main_footer} viewOnly={true}/>
```
Comment (marlonkeating) [inline common/components/componentsBySection/CreateEvent/EventPreviewForm.jsx:29]: Why is this change here?

File: civictechprojects/static/css/partials/_AllowMarkdown.scss [hunk 3]
```diff
@@ -7,4 +7,4 @@
 
 .markdown-container ol:first-child, .markdown-container ul:first-child {
   padding-top: 1rem;
-}
\ No newline at end of file
+}
```
Comment (marlonkeating) [inline civictechprojects/static/css/partials/_AllowMarkdown.scss:10]: Unneeded change

File: civictechprojects/static/css/styles.scss [hunk 4]
```diff
@@ -103,6 +103,5 @@
 @import "partials/JoinConference";
 @import "partials/Terms";
 @import "partials/Privacy";
-@import "partials/_LatestBlogPosts";
 @import "partials/AllowMarkdown";
-@import "partials/RecentProjects";
\ No newline at end of file
+@import "partials/RecentProjects";
```
Comment (marlonkeating) [inline civictechprojects/static/css/styles.scss:107]: Unneeded change

**Review by marlonkeating — CHANGES_REQUESTED** [review]

**Review by marlonkeating — CHANGES_REQUESTED** [review]
To make a clean PR and omit unnecessary commits, I would either make a new branch based off the latest master branch, or do `git rebase master`

**Comment by andrewindechemain** [comment]
> To make a clean PR and omit unnecessary commits, I would either make a new branch based off the latest master branch, or do `git rebase master`

Hi Marlon,I have opened a new branch.The rebase is up to date with Master. The link is : https://github.com/DemocracyLab/CivicTechExchange/tree/Update_ParticipantCount

**Pull request closed without merging**